src/main/java/bio/terra/app/model/AzureRegion.java

```diff
@@ -77,6 +80,9 @@
 
     public static final AzureRegion DEFAULT_AZURE_REGION = AzureRegion.CENTRAL_US;
 
+    public static final List<String> SUPPORTED_REGIONS =
+        Arrays.stream(AzureRegion.values()).map(AzureRegion::toString).collect(Collectors.toUnmodifiableList());
```

Comment:
minor improvement (?)
```suggestion
        Arrays.stream(values()).map(AzureRegion::toString).collect(Collectors.toUnmodifiableList());
```